scripts/vaccination_alerts/utils.ts

```diff
@@ -141,3 +156,30 @@ export function getUserLocationsToAlert(
     .intersection(locationsToAlert)
     .value();
 }
+
+/**
+ * Return a HTML string with the content of the email
+ */
+function renderEmail(data: VaccinationEmailAlertData) {
+  return emailTemplate({ ...data, colors: COLOR_MAP });
+}
+
+/**
+ * Generates the information that the email service needs to send the email
+ */
+export function generateEmailData(emailAddress: string, fipsCode: string) {
+  const emailData = getEmailAlertData(fipsCode);
+  const emailHtmlContent = renderEmail(emailData);
+
+  // Writing the file locally is useful while styling
+  // fs.writeFileSync('alert.html', emailHtmlContent);
+
+  return {
+    Subject: emailData.emailSubjectLine,
+    To: [emailAddress],
+    Html: emailHtmlContent,
+    From: 'Covid Act Now Alerts <noreply@covidactnow.org>',
+    ReplyTo: 'noreply@covidactnow.org',
+    Group: 'VACCINATION_ALERTS',
```

Comment:
The alert emails include the date here which lets us calculate open/click rate for each individual alerts push. We should probably do the same to be consistent? 

https://github.com/covid-projections/covid-projections/blob/b969ebec3def3b48abf00d8262ddcdcaae3ea82a/scripts/alert_emails/utils.ts#L89